src/Extractors/PhpCode.php

```diff
@@ -31,6 +31,7 @@ class PhpCode extends Extractor implements ExtractorInterface
             'np__' => 'npgettext',
             'dnpgettext' => 'dnpgettext',
             'dnp__' => 'dnpgettext',
+            'noop' => 'noop',
         ],
```

Comment:
We have to include both functions: the Translator method `noop` and the global function (`noop__`)